src/custom-element.js

```diff
@@ -793,10 +800,15 @@ export function createAmpElementProto(win, name, opt_implementationClass) {
       this.readyState = 'complete';
       this.layoutCount_++;
       this.toggleLoading_(false, /* cleanup */ true);
-      if (this.layoutCount_ == 1) {
```

Comment:
👍. Without @dvoytenko to rule here, I think I'd rather keep track of two separate ints for now. We can create a separate issue to merge them once he decides.